### PR TITLE
Updated contact thanks formatting

### PR DESCRIPTION
**Pull request by localastronaut**

### CODE REVIEW

File: templates/registration/contact_thanks.html [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 {% block content %}
     <h1>Contact</h1>
+    <div class="contact-thanks">
     <p>Thank you! Your message was received successfully.</p>
```
Comment (RevolutionTech) [inline templates/registration/contact_thanks.html]: Would you mind indenting this line?

Comment (localastronaut) [inline templates/registration/contact_thanks.html]: Did this updated commit ever go through? Thought I had pushed it

Comment (RevolutionTech) [inline templates/registration/contact_thanks.html]: Nope, this PR currently has only one commit.

File: templates/registration/contact_thanks.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
     <h1>Contact</h1>
-    <div class="contact-thanks">
+        <div class="contact-thanks">
     <p>Thank you! Your message was received successfully.</p>
```
Comment (RevolutionTech) [inline templates/registration/contact_thanks.html]: Nope, this isn't right. The `<div>` was fine. You wanted to indent the `<p>`.

Comment (localastronaut) [inline templates/registration/contact_thanks.html]: Shoot. Misread your last comment. Thought the plus in front of it meant that was the line you wanted to indent.

**Comment by RevolutionTech** [comment]
Boom! Looks good. Now you just need to resolve the conflicts on this branch.

File: templates/registration/contact_thanks.html (hunk 2 above)
Comment (RevolutionTech) [inline templates/registration/contact_thanks.html]: Oh, haha. Nope. The plus sign comes from Git- it's just showing that the line was added.

**Pull request merged**